repository-data/webfiles/src/main/resources/site/apicatalogue/apicatalogue.js

```diff
@@ -0,0 +1,53 @@
+const nhsdHiddenClass = 'nhsd-!t-display-hide';
+
+function getAllSearchResults() {
+    return [...document.querySelectorAll('[data-api-catalogue-entry]')];
+}
+
+function getAllSections() {
+    return [...document.querySelectorAll('#list-page-results-list > div')];
```

Comment:
`data-filter-results-item` seems to not be used for anything and could be used here except there are two of them per group. If that's the case, would it be possible to remove this from the one you don't want and use it instead of this relative selector? Just in case the markup changes relative selectors are likely to break.

Reply:
have updated :+1: